datafeeds/resource_library_parser.py

```diff
@@ -0,0 +1,61 @@
+import re
+
+from datafeeds.parser_base import ParserBase
+from helpers.youtube_video_helper import YouTubeVideoHelper
+
+from BeautifulSoup import BeautifulSoup
+
+
+class ResourceLibraryParser(ParserBase):
```

Comment:
Let's add tests for this class - save a copy of the page's html in the `test_data` directory and validate what it spits out.